extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamInfoItemExtractor.java

```diff
@@ -172,6 +172,11 @@ public String getUploaderAvatarUrl() throws ParsingException {
                     .getObject(0).getString("url");
         }
 
+        if(videoInfo.has("channelThumbnail")) {
```

Comment:
Also in the statement above
```suggestion
        if (videoInfo.has("channelThumbnail")) {
```

Reply:
Done!